spec/javascripts/PaymentGateways/stripe/components/StripeCardForm.spec.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { StripeElementsForm } from 'PaymentGateways/stripe/components/StripeFormWrapper'
+import { mount } from 'enzyme'
+
+const defaultProps = {
+    stripePublishableKey: 'abcd',
+    setupIntentSecret: 'efgh',
+    billingAddressDetails: {
+        line1: '1002 Avenue de los Mexicanos',
+        line2: '',
+        city: 'South Park',
+        state: 'CO',
+        postal_code: '80440',
+        country: 'US'
+    },
+    successUrl: '/Broflovski/Residence'
+}
+
+// NOTE: we mount StripeElementsForm instead of the StripeCardForm directly, because of the following requirement from Stripe:
```

Comment:
Regarding StripeFormWrapper.spec.jsx I would also rename it to StripeElementsForm.spec.jsx, because having "-wrapper" files doesn't make sense and they will be refactor in near future.

As for StripeCardForm, to test it without the Context, `'@stripe/react-stripe-js'` needs to be mocked. There are some examples of `jest.mock` in other tests.